schemas/2.0.0.json

```diff
@@ -703,6 +703,14 @@
                     "type": "object",
                     "additionalProperties": false,
                     "properties": {
+                      "name": {
+                        "type": "string",
+                        "description": "Name of the message example."
```

Comment:
can you please specify it should be machine readable name

Reply:
Sure. Pushing an update.